client/src/CustomComponents/CustomText.tsx

```diff
@@ -0,0 +1,17 @@
+import { StyleProp, Text , TextStyle } from 'react-native'
+import React from 'react'
+import { CustomTextProps } from '../types/CustomTextType'
+
+const CustomText : React.FC<CustomTextProps> = ({children , style , fontSize , fontWeight}) => {
+  const basicStyles : StyleProp<TextStyle> = {
+    fontWeight,
+    fontSize
+  };
+  const finalizedStyles  =  {...basicStyles,...style as object};
```

Comment:
for typescript we don't use object at all

Reply:
DONE ✅